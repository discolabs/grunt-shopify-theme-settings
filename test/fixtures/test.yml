---
- section: Appearance & Fonts
  subsections:

    - subsection: Background
      fields:

        - field: Background Style
          name: background_style
          type: select
          options:
            - value: none
              label: None
            - value: color
              label: Custom Color
            - value: image
              label: Custom Image

        - field: Background Color
          name: background_color
          type: color

        - field: Background Image
          name: background_image.jpg
          type: file
          help: Upload a jpeg.

        - field: Repeat Backgound Image?
          name: background_image_repeat
          type: checkbox