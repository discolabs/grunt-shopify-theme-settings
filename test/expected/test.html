
<fieldset>
  <legend>Appearance & Fonts</legend>


  <h3>Background</h3>
  <table>

    <tr>
      <th>
        <label for="background_style">Background Style</label>
      </th>
      <td>

          <select id="background_style" name="background_style">

  <option value="none">None</option>

  <option value="color">Custom Color</option>

  <option value="image">Custom Image</option>

</select>

      </td>
    </tr>


    <tr>
      <th>
        <label for="background_color">Background Color</label>
      </th>
      <td>

          <input id="background_color" name="background_color" class="color" type="text" value="" />

      </td>
    </tr>


    <tr>
      <th>
        <label for="background_image.jpg">Background Image</label>
      </th>
      <td>

          <input type="file" id="background_image.jpg" name="background_image.jpg"   />

      </td>
    </tr>

    <tr>
      <th></th>
      <td><small>Upload a jpeg.</small></td>
    </tr>


    <tr>
      <th>
        <label for="background_image_repeat">Repeat Backgound Image?</label>
      </th>
      <td>

          <input type="checkbox" id="background_image_repeat" name="background_image_repeat" value="1" />

      </td>
    </tr>


  </table>

</fieldset>
